lib/lotus/repository.rb

```diff
@@ -671,12 +672,30 @@ def _persisted?(entity)
         !!entity.id
       end
 
-      def _update_timestamps(entity)
+      # Add time create an entity
+      def _update_created_at(entity)
         if entity.class.attributes.include?(:created_at)
           entity.created_at ||= Time.now.utc
         end
       end
 
+      # Add time update an entity
+      def _update_updated_at(entity)
+        if entity.class.attributes.include?(:updated_at)
+          if entity.updated_at
```

Comment:
`entity.updated_at = entity.updated_at || Time.now.utc`
I think you mean is:
`entity.updated_at = entity.created_at || Time.now.utc`
But The way I see, `entity.updated_at = entity.created_at || Time.now.utc` it won't be appropriate because:
`entity.updated_at = Time.now.utc` if `entity.created_at = nil` but `Time.now.utc` will be assign when we update anything, thereout, `entity.created_at` it can't be nil.
I'll update it to this: `entity.updated_at ? entity.updated_at = Time.now.utc : entity.updated_at = entity.created_at`